### DISCUSSION TITLE
about  Integrate scRNA-seq data and scATAC data.

### DISCUSSION THREAD

**Discussion by woshiyangsi** [GreenleafLab/ArchR#1361] [General]
Hi @rcorces @jgranja24,
According to my annotation results, there are specific clusters in the scATAC data. 
For example, no endothelial cells are detected by scRNA, but endothelial cells are detected by scATAC, so endothelial cells exist specifically in scATAC. 
However, it is found that some cells in scRNA  integrate with this cluster in constrained Integration. In this case, how can I integrate?

Looking forward to your reply,
Siyu.

**Comment by rcorces** [comment]
Minimally, you could create a subsetted ArchR Project using `subsetArchRProject()` and remove the cell types that do not match from the ATAC data before attempting integration.

**Comment by Rui-Jing** [comment]
Hi, Thanks for developing this user-friendly tool. I have the same question. some cells are not detected by scRNA, but these cells are detected by scATAC, so How do I define this group of cells? 

In the '**Defining Cluster Identity with scRNA-seq**' section of the ArchR manual, for unconstrained integration or constrained integration, can the groups without ScRNA be defined according to marker genes first, and then the remaining cells be integrated and annotated by ScRNA? If so, how do I do it
